ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.c

```diff
@@ -30,6 +30,17 @@
 #include <string.h>
 
 #include "tensor_filter_tensorflow_lite_core.h"
+#include "tensor_common.h"
+
+/**
+ * @brief nnapi hw type string
+ */
+static const char *nnapi_hw_string[] = {
+  [NNAPI_CPU] = "cpu",
+  [NNAPI_GPU] = "gpu",
+  [NNAPI_NPU] = "npu",
+  [NNAPI_UNKNOWN] = "unknown"
```

Comment:
Maybe ```[NNAPI_UNKNOWN] = NULL``` to avoid exception case when using find_key_strv()

Reply:
If it is then tensor transform and others should be fixed. I will fix them all later.